src/AaveV3StrategicAssets_20230606/Core.sol

```diff
@@ -0,0 +1,13 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.19;
+
+import {OwnableWithGuardian} from 'solidity-utils/contracts/access-control/OwnableWithGuardian.sol';
+
+abstract contract Core is OwnableWithGuardian {
```

Comment:
I'd rename this contract as "Core" is not descriptive

Reply:
Renamed to Common, couldn't think of anything better but happy to change to anything.